policies/access.md

```diff
@@ -0,0 +1,35 @@
+# GitHub Access Management Policy
+
+## Teams, not Individuals
+
+Access to repositories will be granted only to GitHub teams, not to individuals. A repo may have any number of teams added to it, with the appropriate access levels.
```

Comment:
How to we enforce this?

Allstar has a policy to nag if any individuals have write/admin access, we could use this.

Reply:
We already have an auditing tool; see https://github.com/ossf/github-org-access-scraper.